tests/test_env_max_msg_size.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
```

Comment:
The two test scripts could be collapsed, and simply run the unit test executable multiple times and check the result.